src/index.js

```diff
@@ -100,23 +100,24 @@ export class FoldablesFeature {
       ];
     }
 
+    // The fold is defined as a segment here because it's used in the css spaning polyfill.
     if (this.spanning === "single-fold-horizontal") {
       const screenCenter = (window.innerHeight - this.browserShellSize) / 2;
       const width = window.innerWidth;
       return [
-        { top: 0, left: 0, width, height: screenCenter - this.foldSize / 2 },
-        { top: screenCenter - this.foldSize / 2, height: this.foldSize, left: 0, width },
-        { top: screenCenter + this.foldSize / 2, left: 0, width, height: window.innerHeight }
+        { top: 0, left: 0, width: width, height: screenCenter - this.foldSize / 2 },
+        { top: screenCenter - this.foldSize / 2, height: this.foldSize, left: 0, width: width },
+        { top: screenCenter + this.foldSize / 2, left: 0, width: width, height: screenCenter - this.foldSize / 2 }
       ];
     }
 
     if (this.spanning === "single-fold-vertical") {
       const width = window.innerWidth / 2 - this.foldSize / 2;
       const height = window.innerHeight;
       return [
-        { top: 0, left: 0, width, height},
-        { top: 0, height, left: width, width: this.foldSize },
-        { top: 0, left: window.innerWidth / 2 + this.foldSize / 2, width, height }
+        { top: 0, left: 0, width: width, height: height},
+        { top: 0, height: height, left: width, width: this.foldSize },
```

Comment:
height is the same as height:height - is a JS feature

Reply:
didn't work quite well here...

Reply:
Nevermind, it works.